no function or associated item named `from_fd` found for struct `TunTapInterface` in the current scope

```rust
 $ cargo build --release      
    Updating crates.io index
   Compiling tun2proxy v0.1.1 (/home/acheong/workspace/tun2proxy)
error[E0599]: no function or associated item named `from_fd` found for struct `TunTapInterface` in the current scope
   --> src/tun2proxy.rs:271:34
    |
271 | ...        TunTapInterface::from_fd(*fd, Medium::Ip, options.mtu.unwrap...
    |                             ^^^^^^^ function or associated item not found in `TunTapInterface`

warning: unused import: `std::os::unix::io::AsRawFd`
  --> src/tun2proxy.rs:20:5
   |
20 | use std::os::unix::io::AsRawFd;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: `#[warn(unused_imports)]` on by default

For more information about this error, try `rustc --explain E0599`.
warning: `tun2proxy` (lib) generated 1 warning
error: could not compile `tun2proxy` due to previous error; 1 warning emitted
```

```bash
 $ rustc --version
rustc 1.68.2 (9eb3afe9e 2023-03-27)
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -20,7 +20,7 @@ log = "0.4"
 mio = { version = "0.8", features = ["os-poll", "net", "os-ext"] }
 nix = { version = "0.26", features = ["process", "signal"] }
 prctl = "1.0"
-smoltcp = { version = "0.9", git = "https://github.com/smoltcp-rs/smoltcp", features = ["std"] }
+smoltcp = { version = "0.9.1", git = "https://github.com/smoltcp-rs/smoltcp", features = ["std", "phy-tuntap_interface"] }
 thiserror = "1.0"
 url = "2.3"
 

